### DISCUSSION TITLE
Unable to create ggml model using convert-pt-to-ggml.py script

### DISCUSSION THREAD

**Discussion by ShobhitPanwar** [ggml-org/whisper.cpp#1469] [General]
I followed the instructions mentioned in the readme file but I am unable to create a ggml model. 
Following is what I tried:
"python ./models/convert-pt-to-ggml.py ~/.cache/whisper/base.pt  'C:\Users\shobh\Documents\AI-hub\Repositories\whisper' ./models/whisper-base"

And this is the error I am getting:
"Error: failed to load PyTorch model file: ~\.cache\whisper\base.pt"

I checked the files inside ~\.cache\whisper and it is present inside it:
 ls  ~/.cache/whisper/

    Directory: C:\Users\shobh\.cache\whisper

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----        2023-11-03   6:48 AM      145262807 base.pt

**Comment by bobqianic** [comment]
Have you checked the `Hash` value of your `.pt` model weight?
